src/scss/components/_primary-sidebar.scss

```diff
@@ -1,6 +1,7 @@
 .cmp-primary-sidebar {
   background-color: var(--gray-background-color);
   padding: 1rem;
+  margin-inline-start: auto;
   
   @media (min-width: 50rem) {
     flex-basis: 18.75rem;
```

Comment:
One last thing, with certain types of content in the sidebar, it can get squished down pretty small, so I'd recommend adding `flex-shrink: 0` in addition to the `flex-basis` on wider screens.

Reply:
Inside the media query like this? [db6df75](https://github.com/sparkbox/sparkpress-wordpress-starter/pull/120/commits/db6df75d8392436a173f4d2b786c196f0278869f)

Also moving that `margin-inline-start` inside the mq because we don't want the sidebar scooted over for small screens.

Reply:
Yep, perfect! 🚀